docs/upgrade-guides/upgrading-from-v4-to-v5.mdx

```diff
@@ -505,11 +571,43 @@ If you would like to have your JWT return all of the user's organizations, you c
       })
     ```
 - The `createEmail()` method (and as such the `emails` API endpoint) has been removed. For the time being no alternative method will be provided for creating and sending emails directly from Clerk's JavaScript SDKs. If you're currently relying on this functionality and wish to update, please reach out to [Clerk's support](/support).
+- The `clerkClient.__unstable_options` property used to update the internal options was removed. Create a new clerkClient instance using `createClerkClient` instead of mutating the existing. Example:
+
+    ```diff
+    import { createClerkClient } from '@clerk/backend';
+
+    const clerkClient = createClerkClient({ ...options, secretKey: 'secretKey' });
+
+    - clerkClient.__unstable_options.secretKey = 'newSecretKey'
+    + const newClerkClient = createClerkClient({ ...options, secretKey: 'newSecretKey' });
+    ```
+- The `signJwt`, `hasValidSignature`, `decodeJwt`, `verifyJwt`, `verifyToken` signatures have changed. Instead of returning a value or throwing an error those functions will **always** return a `{ data, error }` value. To keep the existing behaviour check the examples:
+
+  ```diff
+  import { signJwt, hasValidSignature, decodeJwt, verifyJwt } from '@clerk/backend/jwt';
+  import { verifyToken } from '@clerk/backend'
+
+  const { data, error } = await signJwt(...)
+  if (error) throw error;
+
+  const { data, error } = await hasValidSignature(...)
+  if (error) throw error;
+
+  const { data, error } = decodeJwt(...)
+  if (error) throw error;
+
+  const { data, error } = await verifyJwt(...)
+  if (error) throw error;
+
+  const { data, error } = await verifyToken(...)
+  if (error) throw error;
+  ```
 
 Some more niche use cases and their changes:
 
 <Accordion titles={["Removal of httpOptions", "Removal of pkgVersion", "Removal of createIsomorphicRequest & buildRequestUrl", "Use .imageUrl for image related responses/resources"]} heading="h4">
   <AccordionPanel>
+    {/* TODO: The buildRequest is not exposed as public API. There is not a alternative at the moment*/}
```

Comment:
@LekoArts i think we should remove the line below. That's why I added the TODO above.

Reply:
We removed functionality so we should document it. The section has more information to explain that now: https://docs-preview-605.clerkpreview.com/docs/upgrade-guides/upgrading-from-v4-to-v5#clerk-backend

Reply:
@LekoArts i saw you updated the text so i will resolve this.